routes/web.php

```diff
@@ -20,6 +20,9 @@
 Route::get('/products', 'ProductController@index')->name('products.index');
 Route::get('/products/create', 'ProductController@create');
 Route::post('products', 'ProductController@store');
+Route::delete('products', 'ProductController@delete');
```

Comment:
Delete route is correct but it is out of scope for this specific branch. We are only adding the edit configuration right? :)